hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/scm/TestGetCommittedBlockLengthAndPutKey.java

```diff
@@ -110,12 +115,21 @@ public void tesGetCommittedBlockLength() throws Exception {
         ContainerTestHelper
             .getPutBlockRequest(pipeline, writeChunkRequest.getWriteChunk());
     client.sendCommand(putKeyRequest);
-    response = ContainerProtocolCalls
-        .getCommittedBlockLength(client, blockID, null);
+    GenericTestUtils.waitFor(() -> {
+      boolean success = true;
```

Comment:
There is no need for a `boolean` variable in this scenario. We can directly return `true` or `false` when needed.

Reply:
Thank @nandakumar131 for review. Changes done.